authorize/grpc.go

```diff
@@ -25,7 +26,19 @@ func (a *Authorize) IsAuthorized(ctx context.Context, in *authorize.IsAuthorized
 		RemoteAddr: in.GetRequestRemoteAddr(),
 		URL:        getFullURL(in.GetRequestUrl(), in.GetRequestHost()),
 	}
-	return a.pe.IsAuthorized(ctx, req)
+	reply, err := a.pe.IsAuthorized(ctx, req)
+	log.Info().
+		// request
```

Comment:
I wish gRPC preserved `ctx` because it would be awesome to log the associated request id here. Maybe we should add it to gRPC directly (in a follow up PR)?

We should look into whatever magic opencensus is doing to preserve their traceid between rpc services.

Reply:
It's certainly possible to add extra data to the context with `SetValue`. gRPC will pass it through. I can do that in a subsequent PR.